fixieai/agents-ts/src/serve.ts

```diff
@@ -0,0 +1,213 @@
+import bodyParser from 'body-parser';
+import bunyan from 'bunyan';
+import bunyanFormat from 'bunyan-format';
+import bunyanMiddleware from 'bunyan-middleware';
+import express from 'express';
+import fs from 'fs';
+import got from 'got';
+import _ from 'lodash';
+import path from 'path';
+import * as tsNode from 'ts-node';
+
+/**
+ * This file can be called in two environmentS:
+ *
+ *    1. From Python: calling the compiled JS. (This is the normal case.)
+ *    2. From ts-node (This is for local dev.)
+ *
+ * In both cases, we want to be able to `require` an Agent written in TS.
+ * In case (1), we need to call tsNode.register() to enable that.
+ * In case (2), we don't need to call tsNode.register(), because we're already in ts-node. And if we do call it, it
+ * actually creates problems.
+ */
+// @ts-expect-error
+if (!process[Symbol.for('ts-node.register.instance')]) {
+  tsNode.register();
+}
+
+/**
+ * This should be kept in sync with the `AgentConfig` dataclass.
+ */
+export interface AgentConfig {
```

Comment:
We don't actually need all these fields from TS, so arguably I could simplify this type.